app/helpers/lit/frontend_helper.rb

```diff
@@ -15,6 +16,17 @@ def translate(key, options = {})
       def t(key, options = {})
         translate(key, options)
       end
+
+      def pluralize_key(key, count)
```

Comment:
it's not that easy - pluralization rules are different for different languages, ie. please take a look here: https://github.com/svenfuchs/rails-i18n/blob/4223a0010e81242c69dfb5cde7b314f79cea2f0b/rails/pluralization/ar.rb

So this code should take under consideration different rules depending on language code.

Reply:
Okay, basically I just need to fix `pluralize_key` :wink: